Fixing rails, adding colliders, setting up audio in Psyche Scene

smcochra1 opened the pull request:
This PR changes the camera rail in the Psyche scene to go around the planet for a better view. It also adds collider boxes that trigger the audio to play when the spaceship hits them. It also changes the BoxColliderTextTrigger script to check if the game object is null to avoid errors.

Update: This PR will also change the moveSpaceship method so the camera rotation works with the new hierarchy

smcochra1 committed:
Fixing rails, adding colliders, setting up audio

This commit changes the camera rail in the Psyche scene to go around the planet for a better view. It also adds collider boxes that trigger the audio to play when the spaceship hits them.

sqiggle commented:
Here's my thoughts on your branch
I think we need to decide if we want the ship to turn to face the way its going, as if it is propelled from the back. It's much harder but feels less weird than moving in a straight line, facing the same direction
I think the speed was too fast for just a bit

smcochra1 committed:
Fix hierarchy in Psyche and attach camera in Mercury

This commit updates the Psyche scene to have the VRSetup and NonVRSetup hierarchies from dev. It also re-adds the Main Camera to the 'spaceship' parameter for the camera rails in the Mercury scene

smcochra1 committed:
Fixing script to rotate spaceship

This commit changes the moveSpaceship method in RailCamera.cs so that the spaceship will rotate as it moves along the camera rail.

sqiggle reviewed (approved):
The spaceship turns well! Great job, Sydney!
